File: src/main/java/com/example/onemoretimejava/arrays/RichestCustomerWealth.java
package com.example.onemoretimejava.arrays;

public class RichestCustomerWealth {
    public static void main(String[] args) {
       /* Input: accounts = [[1,2,3],[3,2,1]]
        Output: 6*/
        int[][] arr = {
                {1, 2,14},
                {3, 2, 1}
        };
        maximumWealth(arr);
    }

    public static int maximumWealth(int[][] accounts) {
        int n = accounts.length;
        int ans = Integer.MIN_VALUE;
        for (int i = 0; i < n; i++) {
            int sum = 0;
            for (int j = 0; j < accounts[i].length; j++) {
                sum = sum + accounts[i][j];
            }
            ans = Math.max(ans, sum);
        }
        System.out.println(ans);
        return ans;
    }
}

File: src/main/java/com/example/onemoretimekotlin/arrays/RichestCustomerWealth.kt
package com.example.onemoretimekotlin.arrays

import kotlin.math.max

fun main(args: Array<String>) {
    /* Input: accounts = [[1,2,3],[3,2,1]]
           Output: 6*/
    val accounts: Array<IntArray> = arrayOf(
            intArrayOf(1, 2, 3),
            intArrayOf(3, 2, 1),
            intArrayOf(4, 5, 6)
    )
    maximumWealth(accounts)

}

fun maximumWealth(accounts: Array<IntArray>): Int {
    val n = accounts.size;
    var max = Int.MIN_VALUE
    for (i in 0 until n) {
        var sum = 0
        for (j in 0 until accounts[i].size) {
            sum += accounts[i][j]
        }
        max = max(max, sum)
    }
    println(max)
    return max
}